.github/workflows/platform-install-advanced.yml

```diff
@@ -18,19 +18,20 @@ jobs:
           HUB_PORT: 4444
     steps:
       - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v4
         with:
-          java-version: 1.8
+          distribution: 'temurin'
+          java-version: '8'
```

Comment:
Note: Using v4 requires to specify distribution along with the version. Reference: https://github.com/actions/setup-java?tab=readme-ov-file#basic-configuration